protocol/rpcconsumer/rpcconsumer_server.go

```diff
@@ -83,22 +84,41 @@ func (rpccs *RPCConsumerServer) ServeRPCRequests(ctx context.Context, listenEndp
 	rpccs.finalizationConsensus = finalizationConsensus
 	rpccs.consumerAddress = consumerAddress
 	rpccs.consumerConsistency = consumerConsistency
+	initialRelays := true
+	rpccs.relaysMonitor = relaysMonitor
+	if RelaysHealthEnable {
```

Comment:
now it relay health is disabled you wont send init relays...

Reply:
also you should read it from ConsumerCmdFlags ..

Reply:
For some reason I have both lol
I'll the non-ConsumerCmdFlags

Reply:
Fixed in 67ef971273bf89efb5dcd2c65808a6ec6225d0aa and 9faec8715f4d9950b16f25524f94484c0d48248c